modules/overview-of-pipeline-runs.adoc

```diff
@@ -18,6 +18,7 @@ A pipeline run can be classified as the following:
 * Scheduled run: A pipeline run scheduled to execute at least once
 * Triggered run: A previously executed pipeline run.
 
+Each triggered pipeline run generates logs pertaining to each pipeline step. To help you troubleshoot and audit your pipelines, you can review and analyze these logs by using the log viewer. From here, you can search for specific log messages, stop or start the log for pipeline runs that are in progress, and download the step logs to your local machine.
```

Comment:
> You can review and analyze logs for each step in a triggered pipeline run. With the log viewer, you can search for specific log messages, stop or start the log for pipeline runs that are in progress, and download the step logs to your local machine.

Suggestion to cut down and avoid 'pertaining to'.

Reply:
fixed.